src/components/profiles/profile-card.vue

```diff
@@ -306,11 +323,19 @@ widget-editable.q-pa-md(:class="{ 'full-width': list, 'cursor-pointer': !editBut
           .text-grey-7.h-b2.q-px-xs {{ voiceTokenPercentage }}
             | %
           .text-grey-7.h-b2 {{ voiceToken.token }}
+
     .col-auto(:class="{ 'col-12': card, 'col-7': isEnroller, 'q-px-xs': card }" v-if="isApplicant")
       .row.items-center.justify-end.full-height
         .col-4(:class="{ 'text-center': card , 'col-12': card, 'q-mt-md': card, 'justify-end flex': $q.screen.gt.md }" v-if="isEnroller")
           q-btn(:style="{ 'border-radius': '50%' }" :disable="!canEnroll" :loading="submittingEnroll" :icon="'fas fa-times'" @click="onRemoveApplicant" color="negative" no-caps rounded unelevated)
           q-btn.q-ml-xs(:style="{ 'border-radius': '50%' }" :disable="!canEnroll" :loading="submittingEnroll" :icon="'fas fa-check'" @click="onEnroll" color="positive" no-caps rounded unelevated)
+
+    //- .col-auto(:class="{ 'col-12': card, 'col-7': isEnroller, 'q-px-xs': card }" v-if="isAdmin")
+    //-   .row.items-center.justify-end.full-height
+    //-     .col-4(:class="{ 'text-center': card , 'col-12': card, 'q-mt-md': card, 'justify-end flex': $q.screen.gt.md }" v-if="isEnroller")
+    //-       q-btn(:style="{ 'border-radius': '50%' }" :disable="!canEnroll" :loading="submittingEnroll" :icon="'fas fa-times'" @click="onRemoveApplicant" color="negative" no-caps rounded unelevated)
```

Comment:
Why is remove applicant commented? We still need to be able to remove applicants...